### DISCUSSION TITLE
Perhaps a daily challenge history can be added.

### DISCUSSION THREAD

**Discussion by LamGC** [ppy/osu#29552] [Ideas] [answered]
Since the launch of the Daily Challenge, I have really enjoyed this feature. It can recommend beatmaps to me in another way. (Especially since I haven't had much time to play recently)

However, I am really curious about which beatmaps and difficulty were chosen for the previous Daily Challenge? Perhaps this can serve as a new way to recommend beatmaps?

Of course, historical rankings can also be displayed!

**Comment by Tenexxt** [comment, marked as answer]
Already anwsered in #29123
I don't remeber the post peppy said this on, but i remeber that he said that this also should be out this/next week

**Reply by LamGC** [reply]
I roughly searched and couldn't find this discussion, so thank you!
